src/utils/arweave/client.go

```diff
@@ -313,23 +313,34 @@ func (self *Client) GetTransactionDataById(ctx context.Context, tx *Transaction)
 	}
 
 	// Data is base64 url encoded
-	buf, err := base64.RawURLEncoding.DecodeString(string(resp.Body()))
+	data, err := base64.RawURLEncoding.DecodeString(string(resp.Body()))
 	if err != nil {
 		return
 	}
 
-	out.Write(buf)
+	out.Write(data)
 	if out.Len() > 0 {
 		// Do the checks and return
-		if out.Len() != int(tx.DataSize.Int64()) {
+		if len(data) != int(tx.DataSize.Int64()) {
```

Comment:
basically the same as in the original code

Reply:
this change is the result of a previous refactoring that has been rolled back
I restored the original code